lib/service/navigation_service.dart

```diff
@@ -23,9 +23,14 @@ import 'package:autonomy_theme/autonomy_theme.dart';
 import 'package:easy_localization/easy_localization.dart';
 import 'package:flutter/material.dart';
 
+// ignore: implementation_imports
```

Comment:
I think you can add this lib into pubspec.yml to remove this line
`overlay_support: ^2.1.0`